Updates to README

psarin opened the pull request:
Updated instructions on installation
Updated instructions on testing
Added section on additional topics / known issues

psarin committed:
Updates to README
Updated instructions on installation
Updated instructions on testing
Added section on additional topics / known issues

cubiclabs commented:
Something that you might not be aware of is that the SessionInitializerFilter was added to Tomcat some time ago, so if you are using that server you do not need to add the additional jar in - you do still need to modify the web.xml file though:
https://dev.lucee.org/t/installing-the-websocket-extension-on-tomcat-got-much-easier/3012

redtopia commented:
Wow... I never saw that post on the Lucee Dev forum. The jar that you're referring to is `servlet-filter-utils-1.1.1.jar` correct?

cubiclabs commented:
Yes, that is correct. If you are using Tomcat you do not need the additional jar. Just add the filter as described in the forum post.

psarin commented:
Good to know. I will update the document and re-push.

psarin committed:
Remove unneeded step

https://dev.lucee.org/t/installing-the-websocket-extension-on-tomcat-got-much-easier/3012

cubiclabs reviewed (commented):
Note that in the forum post about this, the filter class is different:

`org.apache.catalina.filters.SessionInitializerFilter`

instead of

`net.twentyonesolutions.servlet.filter.HttpSessionInitializerFilter`

This will be **Tomcat only**. I presume that for Jetty or other servers you will still need to add the jar and the original filter class. Not sure how best to explain that concisely in this readme though.

psarin committed:
Fix the filter-class for Tomcat
Add note about non-Tomcat servlet filter

psarin commented:
@isapir Any comments on ths pull request? Do you know if / when it will get accepted / merged? Thanks!

SebastianZ commented on README.md:
```diff
@@ -1,79 +1,204 @@
 
----
+>&nbsp;<br/>
+>💁🏼‍♀️ &nbsp; Watch the [Chart Server Example](https://www.youtube.com/watch?v=rvB7PcNylVY) tutorial.
```
That should read "Chat Server Example".

psarin commented:
@isapir Are you considering this project dead? :)
